typescript/utils/src/env.ts

```diff
@@ -7,3 +7,10 @@ export function safelyAccessEnvVar(name: string) {
     return undefined;
   }
 }
+
+export function envVarToBoolean(value: any) {
```

Comment:
should this be zod?

Reply:
I'll double check but not sure zod is a good fit for type coercion like this